admin/tests/unit/controllers/forms-xml.js

```diff
@@ -177,7 +177,9 @@ describe('FormsXmlCtrl controller', () => {
         .upload()
         .then(() => digest())
         .then(() => {
-          expectStatusError('Upload failed: Unexpected token o in JSON at position 1');
+          expectStatusError(
+            'Upload failed: Unexpected token &#39;o&#39;, &quot;not a valid&quot;... is not valid JSON'
```

Comment:
This error message is bubbling up from the system level.  With the old message, this test actually failed for me locally (running 20.04).  The new version of this message causes the test to pass both locally and on the CI run (with 22.04).

Reply:
I strangely had the same behavior locally, nice catch!